Source/ui_unix/settingsdialog.cpp

```diff
@@ -0,0 +1,88 @@
+#include "settingsdialog.h"
+#include "ui_settingsdialog.h"
+
+#define PREFERENCE_UI_SHOWFPS "ui.showfps"
```

Comment:
The preference defines specific to this port should probably be moved to a common header since they are also used in `mainwindow.cpp`. Not sure we need a "showfps" setting, it should probably be always on. The "showvirtualpad" preference should also be removed since there's no virtual pad on Linux.